From f36ee4d15afd5eafcea3ee856df5f4328df22f72 Mon Sep 17 00:00:00 2001
From: Viacheslav Bocharov <adeep@lexina.in>
Date: Thu, 22 Dec 2022 12:23:35 +0300
Subject: [PATCH 10/12] board: amlogic: jethub j100: add rescue boot from
 microSD

The new JetHub D1+ has a microSD slot. Add resque boot from microSD

Signed-off-by: Viacheslav Bocharov <adeep@lexina.in>
---
 include/configs/jethub.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/jethub.h b/include/configs/jethub.h
index 35f85095ac..e22db4991d 100644
--- a/include/configs/jethub.h
+++ b/include/configs/jethub.h
@@ -12,6 +12,7 @@
 #define BOOTENV_DEV_RESCUE(devtypeu, devtypel, instance) \
 	"bootcmd_rescue=" \
 		"if gpio input 10; then " \
+		"run bootcmd_mmc0; " \
 		"run bootcmd_usb0;" \
 		"fi;\0"
 #else
-- 
2.34.1

